Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.yml

```diff
@@ -89,6 +90,57 @@ configuration:
   - wildfire-virus
   required: false
   type: 16
+- display: Severity of alerts to fetch (XDR)
+  name: xdr_severity
+  defaultvalue: ""
+  type: 16
+  required: false
+  options:
+  - all
+  - Info
+  - Low
+  - Medium
+  - High
```

Comment:
Reverse the order here. High at the top as we do with the other severity drop downs.